routes/auth.js

```diff
@@ -7,21 +7,49 @@ var AuthController = require('../services/controllers/authController');
 var allowOnly = require('../services/routesHelper').allowOnly;
 var UserController = require('../services/controllers/userController');
 var AdminController = require('../services/controllers/adminController');
+var contentService = require('../services/get-view-content');
     
+const customers = require('../services/customer/profile/customerprofile');
 
 var APIRoutes = function(passport) {
-    
-    //router.get('/profile', passport.authenticate('jwt', { session: false }), allowOnly(config.accessLevels.user, UserController.index));
+    let auth = passport.authenticate('jwt', { session: false });
 
-    //router.get('/admin', passport.authenticate('jwt', { session: false }), allowOnly(config.accessLevels.admin, AdminController.index));
+    router.get('/profile', passport.authenticate('jwt', { session: false }), allowOnly(config.accessLevels.user, UserController.index));
 
     router.post('/signup', AuthController.signUp);
     
     router.post('/authenticate', AuthController.authenticateUser);
 
-    return router;
+    router.get('/get-view-content', contentService.getList); 
+    
+    // Retrieve all Customer
+    router.get('/customer', customers.findAll);
+   
+    // Retrieve a single Customer by Id
+    router.get('/customer/:id', auth, customers.findById);
+    //router.get('/profile', passport.authenticate('jwt', { session: false }), allowOnly(config.accessLevels.user, UserController.index));
 
+    // Update a Customer with Id
+    router.put('/customer', customers.update);
+    
+    return router;
+    
 };
 
 module.exports = APIRoutes;
 
+//Use this to diagnose problems with Passport and JWT
```

Comment:
@jmilleriec  We should delete this...

Reply:
Yeah, we certainly can, its a piece of code i found that was super helpful in diagnosing some of the issues with jwt and passport...